SparkPost/2web2ui
# AC-1168: Added link to external docs

### What Changed
 - Added link to external docs in onboarding checklist
 - Checklist item after clicking on link to the docs

### How To Test
 - Go to the dashboard with UI feature flag `messaging_onboarding` enabled
 - Verify under "Let's Code" checklist that copy is correct (see [AC-1168](https://jira.int.messagesystems.com/browse/AC-1168))
 - Verify link works correctly
 - Verify that check appears after clicking link and persists
